Yoshi/Yoshi/View Controllers/Debug Table View Controller/DebugTableViewController.swift

```diff
@@ -50,12 +50,28 @@ extension DebugTableViewController: UITableViewDataSource {
 extension DebugTableViewController: UITableViewDelegate {
 
     func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
-        guard let selectedItem = yoshiTableViewMenu?.displayItems[indexPath.row] else {
+        guard let selectedItem = yoshiTableViewMenu?.displayItems[indexPath.row] where !selectedItem.selected else {
+            navigationController?.popViewControllerAnimated(true)
             return
         }
 
+        // Deselect all items that can not be selected at the same time
+        for item in yoshiTableViewMenu?.displayItems ?? [] {
+            item.selected = false
+        }
```

Comment:
you may be able to replace this for loop with `yoshiTableViewMenu?.displayItems.forEach { $0.selected = false }` to avoid the unwrapping / nil check